lsp-methods.el

```diff
@@ -113,6 +113,12 @@
   ;; single argument, the deserialized response parameters.
   (response-handlers (make-hash-table :test 'eql) :read-only t)
 
+  ;; ‘action-handlers’ is a hash table mapping action to a handler function. It
+  ;; can be used in `lsp-execute-code-action' to determine whether the action
+  ;; current client is interested in executing the action instead of sending it
+  ;; to the server.
+  (action-handlers (make-hash-table :test 'equal) :read-only t)
```

Comment:
Could you make this the last field of the struct? Otherwise, we'd be unnecessarily breaking binary compatibility.

Reply:
Sure, I didn't know that this will have an effect on binary compatibility, thanks!